fury/window.py

```diff
@@ -1146,3 +1146,107 @@ def enable_stereo(renwin, stereo_type):
         stereo_type = 'horizontal'
 
     renwin.SetStereoType(stereo_type_dictionary[stereo_type])
+
+
+def test_and_extract_gl_state(func):
```

Comment:
Can you just rename it `extract_gl_state`

Reply:
@Garyfallidis  and @filipinascimento  asked to me to remove this decorator behavior.

Reply:
ok